regression/cbmc/object-size-01/main.c

```diff
@@ -0,0 +1,17 @@
+#include <assert.h>
+#include <stdlib.h>
+
+void main()
+{
+  char *p = malloc(1);
+  free(p);
+
+  assert(__CPROVER_OBJECT_SIZE(p) == 1);
```

Comment:
:question: Should you also `assert(... != 1` and get a failed for that too?

Reply:
Added